### PR TITLE
Add Jiajiayue Group to Karmada Adopters

### PR DESCRIPTION
**Pull request by xuqianjins**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by RainbowMango — COMMENTED** [review]
Thanks a ton to @xuqianjins ! That means a lot to the community.

By the way, we can leave it alone for `v1.3/v1.4/v1.5` releases. We usually maintain the latest version. No big deal, it depends on you.

File: docs/casestudies/adopters.md [hunk 1]
```diff
@@ -11,19 +11,20 @@ To join this list, please follow [these instructions](https://github.com/karmada
...
+| Ecloud                    | https://ecloud.10086.cn/      | Use Karmada to build a steady and elastic multi-cluster platform for managing workloads | TBD                                                          |
+| Huawei                    | https://www.huaweicloud.com/  | Built the [Ubiquitous Cloud Native Service](https://support.huaweicloud.com/intl/en-us/productdesc-ucs/ucs_productdesc_0001.html) that run applications across regions and clouds with intelligent traffic distribution | TBD                                                          |
+| Hurricane Engines         | https://orang.cloud/          | The [ORANG](https://orang.cloud/) `low-code` platform provides the service for users to build powerful apps quickly with zero lines of code, [ORANG](https://orang.cloud/) uses Karmada to deploy apps across multi-cloud | [Karmada in AIML INSTITUTE](ci123.md)                        |
+| Jiajiayue Group           | http://www.jiajiayue.com.cn   | Design a cross-cloud, cross-region, and cross-zone backup and disaster recovery solution based on Karmada. | TBD                                                          |
```
[the remark this answers is not in the document]
Comment (xuqianjins) [reply docs/casestudies/adopters.md:22]: OK, thanks!

File: i18n/zh/docusaurus-plugin-content-docs/current/casestudies/adopters.md [hunk 2]
```diff
@@ -12,19 +12,20 @@ title: Karmada 生产案例汇总
...
-| 中国联通数字科技   | http://www.chinaunicom.com.cn | 使用 Karmada 构建多策略和多地域的智能分布式容器平台                                                                 | 待定                                                |
\ No newline at end of file
+| 组织/公司名称    | 网站                          | 使用场景                                                     | 案例研究                                                     |
+| ---------------- | ----------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| 道客网络科技     | https://www.daocloud.io/      | 使用 Karmada 构建更灵活、更有弹性的混合多云平台              | [DaoCloud结合Karmada打造新一代企业级多云平台](daocloud.md)   |
+| 华为             | https://www.huaweicloud.com/  | 构建华为云分布式云原生产品 [UCS](https://www.huaweicloud.com/product/ucs.html) 为企业提供云原生业务部署、管理、应用生态的全域一致性体验 | 待定                                                         |
+| 家家悦集团       | http://www.jiajiayue.com.cn   | 基于 Karmada 设计跨云、跨地域、跨可用区的备份容灾解决方案    | 待定                                                         |
```
[the remark this answers is not in the document]
Comment (xuqianjins) [reply i18n/zh/docusaurus-plugin-content-docs/current/casestudies/adopters.md:19]:
```suggestion
| 家家悦集团       | http://www.jiajiayue.com.cn   | 使用 Karmada 构建基于多云、多地域、多可用区集群的统一应用管理平台，实现应用的高可用性和容灾。    | 待定                                                         |
```

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

Thanks!

**Pull request merged**